src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -114,6 +117,26 @@
     @Nullable
     OffsetDateTime getTimeBoosted();
 
+    /**
+     * The time this Member will be released from time out.
+     * <br>If this Member is not in time out, this returns {@code null}.
```

Comment:
```suggestion
     * <br>If this Member is not in time out, this returns {@code null}.
     * This may also return dates in the past, in which case the time out has expired.
```